packages/dashboard/src/components/DashboardFrame/Navigation.tsx

```diff
@@ -23,15 +23,34 @@ function ListItemLinkUnStyled({ nested, ...props }: LinkProps & ListItemProps &
     return <MuiListItem button component={Link} selected={!!useRouteMatch(props.to)} {...props} />
 }
 
-const ListItemLink = styled(ListItemLinkUnStyled)(({ theme, nested }) => ({
-    [`&.${listItemClasses.root}`]: {
-        paddingLeft: nested ? theme.spacing(9) : theme.spacing(2),
-    },
-    [`&.${listItemClasses.selected}`]: {
-        backgroundColor: 'transparent',
-        borderRight: '4px solid ' + (theme.palette.mode === 'light' ? theme.palette.action.selected : 'white'),
-    },
-}))
+const ListItemLink = styled(ListItemLinkUnStyled)(({ theme, nested }) => {
+    const highlightColor = theme.palette.mode === 'light' ? theme.palette.primary.main : '#fff'
+    return {
+        [`&.${listItemClasses.root}`]: {
+            color: theme.palette.mode === 'light' ? '' : 'rgba(255,255,255,.8)',
+            paddingLeft: nested ? theme.spacing(9) : theme.spacing(2),
+        },
+        [`&.${listItemClasses.selected}`]: {
+            color: highlightColor,
+            backgroundColor: 'transparent',
+            position: 'relative',
+            '.MuiListItemIcon-root': {
```

Comment:
please use `listItemIconClasses.root` to replace this,  This current practice is not a strong type